regress/expected.output

```diff
@@ -34,9 +34,21 @@ Loading /tests/test-024.c -> Ok.
 Loading /tests/test-025.c -> Ok.
 Loading /tests/test-026.c -> Ok.
 Loading /tests/test-027.c -> *Too deep recursion
-Loading /tests/test-028.c -> Ok.
```

Comment:
This implies there's a test-028, but it was never checked into source control. I left a spot for it and used -029 instead.